### PR TITLE
[DOC] Reorder the attributes file for easier comparisons

### PR DESCRIPTION
**Pull request by PaulRMellor**
### Type of change

_Select the type of your PR_

- Documentation

Changed the order of the attributes so that we can compare more easily.

_Please describe your pull request_

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by PaulRMellor** [comment]
> Is there some way to avoid the _upstream_ word? ;-)

'Exclusive to Strimzi' might do it. I'll update.

File: documentation/common/attributes.adoc [hunk 1]
```diff
@@ -64,15 +58,12 @@
...
 :ApacheKafkaConnectConfig: link:http://kafka.apache.org/20/documentation.html#connectconfigs[Apache Kafka documentation^]
 :ApacheZookeeperConfig: link:http://zookeeper.apache.org/doc/r3.4.13/zookeeperAdmin.html[Zookeeper documentation^]
 :ApacheKafkaConsumerConfig: link:http://kafka.apache.org/20/documentation.html#newconsumerconfigs[Apache Kafka documentation^]
 :ApacheKafkaProducerConfig: link:http://kafka.apache.org/20/documentation.html#producerconfigs[Apache Kafka documentation^]
-
 :JMXExporter: link:https://github.com/prometheus/jmx_exporter[JMX Exporter documentation^]
-
 :CronExpression: link:http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/tutorial-lesson-06.html[cron expression^]
 
 // Docker image names
```
Comment (laidan6000) [inline documentation/common/attributes.adoc]: I suggest we make this description more generic: "Container image names and repositories".

**Comment by laidan6000** [comment]
I think @ncbaratta should review before we set the `peer review done` label.

File: documentation/common/attributes.adoc (hunk 1 above)
Comment (PaulRMellor) [reply documentation/common/attributes.adoc]: Changed

**Pull request merged**